Лабораторная работа 2/task_1.py

```diff
@@ -0,0 +1,38 @@
+BOOKS_DATABASE = [
+    {
+        "id": 1,
+        "name": "test_name_1",
+        "pages": 200,
+    },
+    {
+        "id": 2,
+        "name": "test_name_2",
+        "pages": 400,
+    }
+]
+
+
+# TODO написать класс Book
+
+class Book:
```

Comment:
нужна документация для класса

Reply:
```suggestion
class Book:
```
"""
    Класс Book представляет книгу с основными атрибутами, такими как: идентификатор, название и количество страниц.

    Атрибуты:
    :param id_ (int): Идентификатор книги
    :param name (str): Название книги
    :param pages (int): Количество страниц в книге

    Методы:
        __str__(): Возвращает строковое представление книги.
        __repr__(): Возвращает официальное строковое представление книги.
    """

Reply:
крутая документация, но только это еще в код добавить)